Library/Homebrew/cmd/update-report.rb

```diff
@@ -69,6 +70,8 @@ def update_report
       updated = true
     end
 
+    migrate_legacy_cache_if_necessary
```

Comment:
Starting fresh screws all our users with bad internet connections. How do you propose we remember?